pinot-controller/src/main/java/com/linkedin/pinot/controller/api/ControllerRestApplication.java

```diff
@@ -106,6 +105,11 @@ protected void configureRouter(Router router) {
 
     attachRoutesForClass(router, PinotVersionRestletResource.class);
 
+    // SegmentCompletionProtocol implementation on the controller
```

Comment:
I will look into it and remove any annotations to make it disappear from swagger